src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -342,6 +343,19 @@ VtolAttitudeControl::Run()
 		return;
 	}
 
+	const hrt_abstime now = hrt_absolute_time();
```

Comment:
Why not put it inside?
```
#if !defined(ENABLE_LOCKSTEP_SCHEDULER)

	const hrt_abstime now = hrt_absolute_time();

	// prevent excessive scheduling (> 500 Hz)
	if (now - _last_run_timestamp < 2_ms) {
		return;
	}

	_last_run_timestamp = now;

#endif // !ENABLE_LOCKSTEP_SCHEDULER
```

Reply:
No real reason, I thought the timestamp might be useful later and didn't want to put more ifdefs in the header.